superset/assets/src/visualizations/deckgl/AnimatableDeckGLContainer.jsx

```diff
@@ -46,7 +58,8 @@ export default class AnimatableDeckGLContainer extends React.Component {
           end={this.props.end}
           step={this.props.step}
           values={this.state.values}
-          onChange={newValues => this.setState({ values: newValues })}
+          range={!this.props.aggregation}
```

Comment:
Use deconstruction from `this.props` to avoid having `this.props.xxx` multiple time.

```
const { end, step, aggregation } = this.props;
const { values } = this.state;
```